src/utils/urls.ts

```diff
@@ -1,9 +1,9 @@
 import isArray from "lodash/isArray";
 import { stringify } from "qs";
 
-export function stringifyQs(params: {}): string {
+export function stringifyQs(params: {}, arrayFormat?: string): string {
   return stringify(params, {
-    arrayFormat: "indices",
+    arrayFormat: arrayFormat || "indices",
```

Comment:
Optional argument added, to change a way how array URLs are passed to apps:
was: list[0]=aaa$list[1]=bbb
now: list=aaa,bbb